mantidproject/mantid
# ConvFit sample log update

Fixes #14267

The output workspace from ConvFit should now contain `sample_filename` and `resolution_filename`

**Data for testing is available from `/ExternalData/Testing/Data/UnitTest/`**
# To Test
- Open ConvFit (Interfaces > Indirect > Data Analysis > ConvFit)
- Sample = `irs26176_graphite002_red.nxs`
- Resolution = `irs26173_graphite002_res.nxs`
- Change Fit Type to `One Lorentzian`
- Check `Temp. Correction` and enter a value into the corresponding input field
- Click `Run`
- Ensure that the OutputWorkspace (ending in `_Result`) contains sample_filename, resolution_filename, temperature_correction and temperature_value.
- Ensure that the workspaces in the group workspace (workspace ending in `_Workspaces`) also has these entries in the log (It is probably only necessary to check one or two of these workspaces)
